pom.xml

```diff
@@ -2218,31 +2190,13 @@
                 <version>${version.org.jboss.byteman}</version>
                 <scope>test</scope>
             </dependency>
-            <dependency>
-                <groupId>org.jmockit</groupId>
-                <artifactId>jmockit</artifactId>
-                <version>${version.org.jmockit}</version>
-                <scope>test</scope>
-            </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-core</artifactId>
                 <version>${version.org.mockito}</version>
                 <scope>test</scope>
```

Comment:
Hi @parsharma , there are still some dependencies in the main pom with the test scope. Why are those not managed via the test bom? You have moved some of them, but not all, so I wonder what's the reason.

Reply:
Hi @yersan,
Other modules use these dependencies, so I have not moved them.
Thanks

Reply:
In that case, the approach should be to add the `testbom` in the dependency management section of the other modules that need these pure test dependencies.

The general idea is to move all the test dependencies outside of the main pom.xml to avoid incorporating pure test dependencies to WildFly from WildFly Core.

Reply:
Yes, I will move above dependencies also. Thanks, @yersan.